helm/logging-app-chart/Chart.yaml

```
@@ -0,0 +1,2 @@
+name: logging-app-chart
+version: 1.0.0-[[ .SHA ]]
```

```suggestion
version: [[ .Version ]]
```

This has changed with App Catalog so I think it should be this.